### ISSUE TITLE
Openai API endpoint location

### ISSUE THREAD

**Issue by matbee-eth** [ollama-webui/ollama-webui#471]
Trying to use this ollama-webui as an openai endpoint, but it doesnt seem to work- trying to use continue.dev with ollama

**Comment by justinh-rahb** [comment]
Hi @matbee-eth, I understand that you're looking for a way to use Ollama's functionality with an OpenAI-compatible API endpoint. However, the Ollama WebUI project is separate and doesn't offer this capability. Instead, I would recommend checking out alternative projects like [LiteLLM](https://github.com/BerriAI/litellm) or [LocalAI](https://github.com/mudler/LocalAI).

Additionally, based on the continue.dev documentation, it seems that it can directly work with Ollama's API without requiring an OpenAI-compatible endpoint. So, you may want to explore this option as well.
https://continue.dev/docs/reference/Model%20Providers/ollama

**Comment by matbee-eth** [comment]
Yeah I use it locally as well, is there an api key system or a way to disable auth for localhost?

**Comment by justinh-rahb** [comment]
To be clear, Ollama WebUI does not provide an API, and the author has stated previously that an option for disabling authentication isn't a goal of this particular project. If you're looking for a lighter-weight version of the application for personal local usage, you can check out [Ollama WebUI Lite](https://github.com/ollama-webui/ollama-webui-lite).

If you're looking for a way to provide OpenAI API and manage API keys for Ollama, LiteLLM would be ideal. I recommend reading their documentation for a thorough understanding of its capabilities.
